src/views/InteractionDesigner.vue

```diff
@@ -2,9 +2,11 @@
   <div
     v-if="activeFlow"
     class="interaction-designer-contents">
-    <tree-builder-toolbar @height-updated="handleToolBarHeightUpdate" />
```

Comment:
Do you know if we need to then update CustomInteractionDesigner at all? 🤷‍♀️

Reply:
I am not sure about this 
@jcbashdown  Any thoughts of whether I need to make this change on `CustomInteractionDesigner` as well?

Reply:
Yes, we'd have to make changes to the interaction designer. Should be fairly simple though? I will look into doing this now in the 4193 PR